src/Auditor.jl

```diff
@@ -124,8 +124,8 @@ function audit(prefix::Prefix, src_name::AbstractString = "";
                 end
             """
             try
-                p = open(`$(Base.julia_cmd()) -e $dlopen_cmd`)
-                wait(p)
+                p = run(`$(Base.julia_cmd()) -e $dlopen_cmd`,
```

Comment:
I've just seen that this at some point this was already a `run`, but it was changed to `open` in https://github.com/JuliaPackaging/BinaryBuilder.jl/commit/a906d86d57a97ca6a84c1ed5336fc11fd89290de (part of #321).  The commit message is "Properly isolate `dlopen()` checks again", even though it's not clear to me why `open` is more isolated than `read`.  The problem with `open` is that I couldn't find any way to make it actually reprint the error message.

Reply:
`run` was introduced in https://github.com/JuliaPackaging/BinaryBuilder.jl/commit/86f81b376169afd21567dacc82ded399f8f23aef, apparently with the goal to make dlopen errors more verbose, goal defeated by the use of `open`.